src/core_codemods/docs/pixee_python_enable-jinja2-autoescape.md

```diff
@@ -0,0 +1,17 @@
+This codemod ensures you configure jinja2 to turn on autoescaping of HTML content. Unfortunately, the jinja2
```

Comment:
```suggestion
This codemod enables autoescaping of HTML content in `jinja2`. Unfortunately, the jinja2
```

Reply:
I'll update it but I thought (idk why) we were moving toward more active and not passive voice.